packages/manager/modules/pci/src/projects/new/payment/components/add/controller.js

```diff
@@ -42,6 +42,13 @@ export default class PciProjectNewPaymentMethodAddCtrl {
     );
   }
 
+  getPayPalChargeAmount() {
+    const uCent = 1000000; // micro cent factor -> 10^-6
+    const priceInCent = this.voucherPlan.pricings[0].price / uCent;
+
+    return priceInCent / 100; // To get the price in currency base
```

Comment:
Could API return a non ucent price ? not the best to always have to make conversion

Reply:
The price is coming from Agora catalog, i am not sure they will able to add the factor.